<?xml version="1.0"?>
<launch>
  <node pkg="prob_rob_labs" type="ekf_localization" name="ekf_localization_2" output="screen">
  </node>
</launch>
